pkgs/applications/science/biology/nf-core/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, click
+, filetype
+, gitpython
+, jinja2
+, jsonschema
+, markdown
+, packaging
+, pdiff
+, pillow
+, pre-commit
+, prompt-toolkit
+, pytest
+, pytest-workflow
+, pyyaml
+, questionary
+# NOTE Not going to support as it requires 10+ packages and is a niche feature
+# refgenie
+, requests
+, requests-cache
+, rich
+, rich-click
+, tabulate
+, trogon
+}:
+
+buildPythonPackage rec {
+  pname = "nf-core";
+  version = "2.12.1";
+  pyproject = true;
+
```

Comment:
The package should also indicate that it requires at least Python 3.8 (e.g. through `pythonOlder`).

Reply:
Also didn't know this was a thing!